proposals/lip-0053.md

```diff
@@ -766,13 +781,13 @@ Those options for partial cross-chain update transactions are not expected to be
 
 #### Invalid Cross-Chain Messages
 
-A cross-chain update transaction may have been created with validly signed information according to the [validity rules](#parameters-validity) and hence be included in the blockchain. However, the transaction could include invalid CCMs. Those will be detected when trying to process the `inboxUpdate`.
+A cross-chain update transaction may have been created with validly signed information according to the validity rules and hence be included in the blockchain. However, the transaction could include invalid CCMs. Those will be detected when trying to process the `inboxUpdate`.
 
-If a CCM is invalid (as specified in the ["Execute Cross-chain Updates" section](#execute-cross-chain-updates)), then the sending chain is [terminated][lip-0045#termination]. The CCU processing then continues and CCMs from other chains are applied. All CCMs from terminated chains have no effect, they are neither applied nor forwarded.
+If a CCM is invalid, then the sending chain is [terminated][lip-0045#termination]. The CCU processing then continues and CCMs from other chains are applied. All CCMs from terminated chains have no effect, they are neither applied nor forwarded.
 
 #### First Cross-chain Update from a Sidechain
```

Comment:
Perhaps also change this title to "first cross-chain messages" from a sidechain. I was also confused in the beginning on this difference, so would be good to make this clear.

Reply:
changed